doc/nrf/ug_bootloader.rst

```diff
@@ -92,7 +92,96 @@ These signed variants can be used to perform an upgrade procedure through the :r
 Adding a bootloader chain to your application
 *********************************************
 
-See :ref:`bootloader_build_and_run` in the documentation of the :ref:`bootloader` sample for instructions on how to add an immutable bootloader as first stage of the secure bootloader chain.
+The |NCS| includes a sample implementation of an immutable bootloader.
+As upgradable bootloader, it provides and extends :doc:`mcuboot:index`.
```

Comment:
This sentence looks strange.

Reply:
@ru-fu Do you have a suggestion here. I don't really know what we want to put here.

Reply:
I don't know where mcuboot:index links, otherwise I could suggest something.

Reply:
```suggestion
As solution for implementing an upgradable bootloader, it provides and extends :doc:`mcuboot:index`.
```
The link goes to https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/mcuboot/index.html

Reply:
Thanks for the link.

It still doesn't make sense to me :P. What does "it" refer to? And what does it mean to "provide and extend" MCUboot?

Reply:
Oh, now I can actually answer ...?!
What I'm trying to state in these two sentences is what solutions the NCS provides for the bootloader chain. For the immutable bootloader, there is a sample. For the upgradable bootloader, there is our fork of MCUboot (which is an extension of the "real" MCUboot).

Reply:
Ok, that makes sense. Like I said, the "it" threw me off. I thought "it" referred to the immutable bootloader.